syntax = "proto3";

message UserRegister {
    string first_name = 1;
    string last_name = 2;
    string email = 3;
    string phone = 4;
    string status = 5;
    string password = 6;
}
